lakefs_provider/example_dags/lakefs-dag.py

```diff
@@ -65,18 +63,12 @@ def check_branch_object(task_instance, repo: str, branch: str, path: str):
     hook = LakeFSHook(default_args['lakefs_conn_id'])
     print(f"Trying to check if the following path exists: lakefs://{repo}/{branch}/{path}")
     try:
-        hook.get_object(repo=repo, ref=branch, path=path)
+        hook.stat_object(repo=repo, ref=branch, path=path)
```

Comment:
AFAIR this was intentionally `get_object`, to test that flow too.  @itaiad200 may have more context.  Of course, right now we only go through `get_object` and never through `stat_object` on the hook.

Please consider reverting this change and adding an issue also to cover `hook.stat_object` in the test.

Reply:
Calling get_object will pull the full object content - if it is for the example sake, maybe we can change the workflow.

Reply:
I love it when I get tagged in code I wrote 2 years ago 😄 I agree that `stat` is more accurate.